src/core/Akka.Streams.Tests/Dsl/AttributesSpec.cs

```diff
@@ -82,7 +81,7 @@ public override IModule WithAttributes(Attributes attributes)
             protected override SinkModule<NotUsed, Task<Attributes>> NewInstance(SinkShape<NotUsed> shape)
                 => new AttributesSink(Attributes, shape);
 
-            public override ISubscriber<NotUsed> Create(MaterializationContext context, out Task<Attributes> materializer)
+            public override object Create(MaterializationContext context, out Task<Attributes> materializer)
```

Comment:
Because that can now also return a Publisher :smile:
See https://github.com/akkadotnet/akka.net/pull/2292/files#diff-a4bb8f441c4b4960d165988a8f8473c4R113